hedera-mirror-rest/utils.js

```diff
@@ -823,6 +823,18 @@ const secNsToSeconds = (secNs) => {
   return math.floor(Number(secNs));
 };
 
+/**
+ * Increment timestamp (nnnnnnnnnnnnnnnnnnn format) by 1 day and 1 second
+ * @return {String} (seconds.nnnnnnnnn format)
+ */
+const incrementTimestampByOneDay = (seconds) => {
+  if (_.isNil(seconds)) {
+    return null;
+  }
+
+  return nsToSecNs(math.add(math.bignumber(seconds.toString()), secNsToNs(86401)));
```

Comment:
Why repeatedly to the conversion.
Just store the ns equivalent of 1day and 1 second and call is here
```suggestion
  return nsToSecNs(math.add(math.bignumber(seconds.toString()), oneDayAndOneNs));
```

Reply:
Great, I have added a constant to the new decrement function.